packages/cosmic-swingset/test/inquisitor.test.ts

```diff
@@ -46,12 +50,15 @@ test('smoke test', async t => {
 
   // Build and exercise the helpers.
   const { db } = swingStore.internal;
-  // @ts-expect-error missing EV
-  const { stable: helpers } = makeHelpers({ db });
+  // TODO: Provide a real EV object if needed for more complex tests
+  const EV =
+    fn =>
+    (...args) =>
+      fn(...args);
+  const { stable: helpers } = makeHelpers({ db, EV });
```

Comment:
This isn't sufficient for use anyway; the only helper that references `EV` is `runCoreEval`, and it does `EV.vat('bootstrap').consumeItem('coreEvalBridgeHandler')` and `EV(coreEvalBridgeHandler).fromBridge(coreEvalAction)`.
```suggestion
  const { stable: helpers } = makeHelpers({ db, EV: undefined });
```